rviz_default_plugins/include/rviz_default_plugins/displays/image/ros_image_texture.hpp

```diff
@@ -127,6 +143,11 @@ class ROSImageTexture : public ROSImageTextureIface
   ImageData setFormatAndNormalizeDataIfNecessary(ImageData image_data);
   void loadImageToOgreImage(const ImageData & image_data, Ogre::Image & ogre_image) const;
 
+  void image_convert_yuv422_to_rgb(uint8_t *dst_img, uint8_t *src_img,
```

Comment:
change the naming style to be similar to theirs by replacing underscore with Caps